airqo-platform/AirQo-api
# Ft device measurements calibrated values

# Add Calibrated Values to device measurements

**_WHAT DOES THIS PR DO?_**
Adds calibrated values to the device measurements. For both Airqo and KCCA

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
None

**_WHAT IS THE LINK TO THE PR BRANCH_**
[Device measurements with calibrated values](https://github.com/airqo-platform/AirQo-api/tree/ft-device-measurements-calibrated-values)

**_HOW DO I TEST OUT THIS PR?_**
Checkout this branch. Run the following to setup a test environment
> `cd AirQo-api/src/data-mgt/python/jobs`
> `python3 -m venv [DESTINATION_FOLDER]`
> `source [DESTINATION_FOLDER]/bin/activate`
> `pip install -r requirements.txt`
> `export DEVICE_REGISTRY_BASE_URL="{DEVICE_REGISTRY_BASE_URL}"`
> `export FEEDS_BASE_URL="{FEEDS_BASE_URL}"`
> `export CALIBRATE_URL="{CALIBRATE_URL}"`
> `export CLARITY_API_KEY="{CLARITY_API_KEY}"`
> `export CLARITY_API_BASE_URL="{CLARITY_API_BASE_URL}"`

To test Airqo 
> `python airqo_device_measurements.py`

To test Kcca 
> `python kcca_device_measurements.py` 

Observe events collection for Airqo and Kcca for the calibrated values. 

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
N/A

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
N/A

**_ARE THERE ANY RELATED PRs?_**
[eliminate duplicate entries for days](https://github.com/airqo-platform/AirQo-api/pull/278)
[null values - Closed, replaced with this one](https://github.com/airqo-platform/AirQo-api/pull/279)
